spec.md

```diff
@@ -0,0 +1,228 @@
+# Open Container Initiative
+
+## Artifacts Specification
+
+This Artifacts Spec provides a reference for artifact authors and registry implementors to support new artifact types with pre 1.0 implementations of the [distribution-spec][distribution-spec]. More particularly this repository (spec) has been tasked by the OCI TOB to serve 3 primary goals:
+
+1. **Artifact Authors** - guidance for authoring new artifact types. Including a clearing house for [well known][def-well-known-types] artifact types.
+1. **Registry Operators and Vendors** - guidance for how operators and vendors can support new artifact types, including how they can opt-in or out of well known artifact types. Registry operators that already implement `mediaType` filtering will not have to change. The artifact repo will provide context on how new `mediaTypes` can be used, and how `mediaTypes` can be associated with a type of artifact.
```

Comment:
Do we want to say "how they can opt-in or provide opt-in support"?

Reply:
I've gone back and forth on this one. I'm now back to a separate `registry-implementors.md` file for how registry implementers can support artifact types. See PR #7 for a previous version. 
I'll rewrite #7 to account for the latest. Trying to put all in this single `spec.md` proved too much content to get merged.

Reply:
Can we link it from here?

Reply:
Once it's done. That was the problem with the first set of PRs. I had it broken up in to several files with separate PRs for reviewing. That was just a git-hell hole I asked everyone to dive into.
My current thinking is once spec.md gets merged, we'll update the `artifact-authors.md` doc for publishing well known types and the `registry-operators.md`. These PRs will have updates to spec.md and readme.md for links to these additional docs. It's a little like "a dimmer switch, not an on switch"